Add Prague 2021 landing page

mxsasha opened the pull request:
Particular points of attention:

- Check the news post and add the right date estimates for ticket sales etc.
- The config file and all the details in there (also needs some dates still filled in)

mxsasha committed:
Add Prague 2021 landing page

mxsasha committed:
Remove elements from config that are empty for now

thatdocslady commented on docs/_data/prague-2021-config.yaml:
```diff
@@ -148,15 +140,6 @@ about:
   projector_ratio: "16:9"
   job_fair_room: "Sponsor area"
 
-cfp:
```
Would deleting these cause us problems later when we rebase against the latest Portland content? Juuuuuuust checking :D

mxsasha replied:
No, we can just add them at the time.

thatdocslady commented on docs/_data/prague-2021-config.yaml:
```diff
@@ -83,14 +83,6 @@ sponsorship:
-  ends:
 
 date: # how do we handle these? human readable would be nice.
   main: "**April 25-27, 2021, online**"
```
```suggestion
  main: "**October 3-5, 2021, online**"
```

thatdocslady commented on docs/_data/prague-2021-config.yaml:
```diff
@@ -0,0 +1,172 @@
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+
+tickets:
```
@ericholscher are we updating these prices?

mxsasha replied:
Set to 150/100/25 as discussed

thatdocslady commented on docs/_data/prague-2021-config.yaml:
```diff
@@ -0,0 +1,172 @@
+    icon: conference
+  # These are listed in the schedule
+  day_one:
+    event: Hike
```
Where is the boat ride? Should we even have it since it's DIY?

mxsasha replied:
This is what we had last year apparently:
https://github.com/writethedocs/www/blob/master/docs/_data/prague-2020-config.yaml#L94-L100

thatdocslady commented on docs/_data/prague-2021-config.yaml:
```diff
@@ -0,0 +1,172 @@
+    summary: This is the main event! Hear from lots of interesting folks about all things documentation. We will have talks all day, and unconference sessions running in parallel.
+    dotw: Monday
+    talk_time: 10:00-18:00
+    unconference_time: 9:45-17:00
```
```suggestion
    unconference_time: 10:45-18:00
```

thatdocslady commented on docs/_data/prague-2021-config.yaml:
```diff
@@ -0,0 +1,172 @@
+    summary: And the conference goes on! We are already starting to miss you. |:heart:|
+    dotw: Tuesday
+    talk_time: 10:00-18:00
+    job_fair_time: 9:45-12:00
```
```suggestion
    job_fair_time: 10:00-12:00
```

thatdocslady replied:
I think since the first talk is a regular talk we can start the job fair at the same time

thatdocslady commented on docs/_data/prague-2021-config.yaml:
```diff
@@ -0,0 +1,172 @@
+    dotw: Tuesday
+    talk_time: 10:00-18:00
+    job_fair_time: 9:45-12:00
+    unconference_time: 14:15-17:00
```
```suggestion
    unconference_time: 15:15-18:00
```

thatdocslady commented on docs/_data/prague-2021-config.yaml:
```diff
@@ -0,0 +1,172 @@
+about:
+  attendees: 600
+  summary:
+    "The main presentation track takes place from **April 26-27 (Monday and Tuesday) from 10:00 to 18:00**.
```
```suggestion
    "The main presentation track takes place on **October 4-5 (Monday and Tuesday) from 10:00 to 18:00**.
```

thatdocslady commented on docs/conf/prague/2021/news/welcome.rst:
```diff
@@ -0,0 +1,66 @@
+
+This is another virtual conference, which means you can attend,
+speak or have a sponsor presence from anywhere in the world.
+The conference will take place during day time in CET_.
```
```suggestion
The conference will take place during the day time in CET_ time zone.
```

thatdocslady commented on docs/conf/prague/2021/news/welcome.rst:
```diff
@@ -0,0 +1,66 @@
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+We plan to launch our CFP in early XXXX and have it last until sometime in XXXX.
```
```suggestion
We plan to launch our CFP in early May and have it last until sometime in June.
```

thatdocslady replied:
Does this timeline make sense? @plaindocs ?

thatdocslady commented on docs/conf/prague/2021/news/welcome.rst:
```diff
@@ -0,0 +1,66 @@
+Tickets
+-------
+
+Tickets will officially go on sale in **XXXXX {{ year }}**.
```
```suggestion
Tickets will officially go on sale in **April {{ year }}**.
```

thatdocslady replied:
What do we think about the ticket sales timeline? @ericholscher ?

thatdocslady commented on docs/conf/prague/2021/sponsors/prospectus.rst:
```diff
@@ -0,0 +1,282 @@
+- A table at our **virtual job fair**
+- Name included in welcome announcement in email newsletters and social media
+
+The **Second Draft** package costs **{{sponsorship.second_draft.price}}**.
```
@ericholscher are we tweaking sponsorship prices this year?

mxsasha replied:
Set to 1,500, 3,000, 6,500, 12,500 as discussed

thatdocslady reviewed (commented):
Looks great! Bunch of nit-picks, and questions about pricing. I itentionally skipped all the pages that are hidden for now, since we'll be rebasing them off of Portland before we launch the full website.

mxsasha committed:
Update docs/_data/prague-2021-config.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

mxsasha committed:
Update docs/_data/prague-2021-config.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

mxsasha committed:
Update docs/_data/prague-2021-config.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

mxsasha committed:
Update docs/_data/prague-2021-config.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

mxsasha committed:
Update docs/_data/prague-2021-config.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

mxsasha committed:
Update docs/conf/prague/2021/news/welcome.rst

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

mxsasha committed:
Fix Prague 2021 templates for unknown YAML info.

This is how our templates should be working already -
the previous behaviour was a bug that we got away with most of the time.

mxsasha commented:
So, removing some items like `cfp` from the YAML (which is correct if we don't have CFP info yet) makes the YAML valid (which will be checked again by CI soon ™️) but broke some templates.

735a2d31c8529802b1edddec30c782daff7af3cb fixes this, and we should retain these fixes when copying templates going forward.

mxsasha committed:
Update timelines and pricing.

mxsasha commented:
I've set the timeline suggested by Mikey and updated pricing. Ready to merge from my view.

ericholscher commented:
We're only releasing the landing pages right? So it won't have any of the pricing details published at this stage, or are we doing the full site? 

I'm 👍 on shipping it -- getting the dates announced is useful as early as possible.

ericholscher commented on docs/conf/prague/2021/schedule.rst:
```diff
@@ -0,0 +1,201 @@
+If you get into town early, join us on the hike and take the chance to explore Portland in all of its glory.
+
+* **Where**: Lower Macleay Park or Macleay Park Entrance.
+* **When**: **{{ date.day_two.hike_time }} {{ tz }}**
```
We already lost the changes I made here. I wonder if we're getting closer to the point we could make this an include, so we stop losing changes across versions?

mxsasha replied:
I will look into this tomorrow.

mxsasha replied:
I have extracted it into a generic schedule for 2021, taken from the Portland schedule you edited. Maybe we get stuck with that later, but I prefer that to copying and pasting and losing changes.

mxsasha commented:
> We're only releasing the landing pages right? So it won't have any of the pricing details published at this stage, or are we doing the full site?
> 
> I'm 👍 on shipping it -- getting the dates announced is useful as early as possible.

You are correct - we got derailed into fixing up pricing because it was part of setting up the YAML.

mxsasha committed:
Extract 2021 schedule pages to generic include

thatdocslady reviewed (approved):
Preview build looks great!

ericholscher commented:
Cool -- let's ship it. Who's in charge of posting on the socials? @thatdocslady?

mxsasha committed:
Enable Prague 2021 feed
